stylus-tools/src/activator.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2025, Offchain Labs, Inc.
+// For licensing, see https://github.com/OffchainLabs/stylus-sdk-rs/blob/main/licenses/COPYRIGHT.md
+
+use crate::call;
+use eyre::Result;
+use std::borrow::ToOwned;
+use typed_builder::TypedBuilder;
+
+/// Defines the configuration for activating a Stylus contract.
+/// After setting the parameters, call `Activator::activate` to perform the activation.
+#[derive(TypedBuilder)]
+#[builder(field_defaults(default, setter(into)))]
+pub struct Activator {
+    #[builder(!default)]
+    rpc: String,
+    dir: Option<String>,
+
+    #[cfg_attr(
+        feature = "integration-tests",
+        builder(default = crate::devnet::DEVNET_PRIVATE_KEY.to_owned())
+    )]
+    #[cfg_attr(not(feature = "integration-tests"), builder(!default))]
```

Comment:
If you follow the approach that I described in the other comment, regarding removing `#[builder(field_defaults(default, setter(into)))]`, you will not need two cfg_attr here.